src/loki/isochrone_action.cc

```diff
@@ -46,6 +64,12 @@ namespace valhalla {
       if (locations.size() > max_locations.find("isochrone")->second)
         throw valhalla_exception_t{400, 150, std::to_string(max_locations.find("isochrone")->second)};
 
+      //check the distances
+      auto max_location_distance = std::numeric_limits<float>::min();
+      check_distance(locations, max_distance.find("isochrone")->second, max_location_distance);
+      if (!healthcheck)
+        valhalla::midgard::logging::Log("max_location_distance::" + std::to_string(max_location_distance * kKmPerMeter) + "km", " [ANALYTICS] ");
```

Comment:
do we need to actually pick this up in logstash @kdiluca ? we dont use that much so maybe its not worth the effort at this point?